docs/UserGuide/Cluster/Cluster-Setup.md

```diff
@@ -143,284 +134,212 @@ Please set the important parameters in iotdb-datanode.properties:
+* Add DataNode
 
-```
-sbin\stop-datanode.bat
-```
-```
-sbin\stop-confignode.bat
-```
-In Linux:
-```
-bash sbin/stop-datanode.sh
-```
-```
-bash sbin/stop-confignode.sh
-```
-Be careful not to miss the "sudo" label, because some port info's acquisition may require root authority. If you can't sudo, just
-use "jps" or "ps aux | grep iotdb" to get the process's id, then use "kill -9 <process-id>" to stop the process.  
+### 6.1.1 Start the Seed-ConfigNode
 
-## Start StandAlone
-If you just want to setup your IoTDB locally, 
-You can quickly init 1C1D (i.e. 1 Confignode and 1 Datanode) environment by our shells.
+The first ConfigNode to start is the Seed-ConfigNode, which marks the creation of the new cluster.
+Open the Seed-ConfigNode configuration file ./conf/iotdb-confignode.properties and set the following parameters:
 
-This will work well if you don't change our default settings.
+| **Configuration**              | **Description**                                                                                    |
+|--------------------------------|----------------------------------------------------------------------------------------------------|
+| cn\_internal\_address          | Internal rpc service address of ConfigNode                                                         |
+| cn\_internal\_port             | Internal rpc service port of ConfigNode                                                            |
+| cn\_consensus\_port            | ConfigNode replication consensus protocol communication port                                       |
+| cn\_target\_config\_node\_list | Set it to its own internal communication address, that is cn\_internal\_address:cn\_internal\_port |
 
-Start on Windows:
-```
-sbin\start-standalone.bat
-```
-Start on Linux:
-```
-bash sbin/start-standalone.sh
+After the configuration is complete, run the startup script:
 ```
+# Linux foreground
+bash ./sbin/start-confignode.sh
 
-Besides, with our shell, you can also directly kill these processes.
+# Linux background
+nohup bash ./sbin/start-confignode.sh >/dev/null 2>&1 &
 
-Stop on Windows:
-```
-sbin\stop-standalone.bat
-```
-Stop on Linux:
-```
-bash sbin/stop-standalone.sh
+# Windows
+.\sbin\start-confignode.bat
 ```
 
-Note: On Linux, the 1C1D processes both launches in the background, and you can see the logs for details. 
+For more details about other configuration parameters of ConfigNode, see the
+[ConfigNode Configurations](https://iotdb.apache.org/UserGuide/Master/Reference/ConfigNode-Config-Manual.html).
 
-The stop-standalone.sh may not work well without sudo, since IoTDB's port numbers may be invisible without permission. 
-If stop-standalone.sh meets some error, you can use "jps" or "ps aux | grep iotdb" to obtain the process ids,
-and use "sudo kill -9 <process-id>" to manually stop the processes.
+### 6.1.2 Add ConfigNode (Optional)
 
-## Start Cli
+You can add more ConfigNodes to the cluster to ensure high availability of ConfigNodes.
```

Comment:
Question: If a user initially wants to deploy 3 confignodes, can all the 3 nodes are seed-confignodes? or the other 2 nodes must be deployed by "Add ConfigNode"??

Reply:
The other 2 ConfigNodes must be deployed by "Add ConfigNode"